console/style.rst

```diff
@@ -140,6 +140,31 @@ Content Methods
             ]
         );
 
+:method:`Symfony\\Component\\Console\\Style\\SymfonyStyle::horizontalTable`
+    It displays the given array of headers and rows as a compact horizontal table::
+
+        $io->horizontalTable(
+            ['Header 1', 'Header 2'],
+            [
+                ['Cell 1-1', 'Cell 1-2'],
+                ['Cell 2-1', 'Cell 2-2'],
+                ['Cell 3-1', 'Cell 3-2'],
+            ]
+        );
+
+:method:`Symfony\\Component\\Console\\Style\\SymfonyStyle::definitionList`
+    It displays the given arguments as compact table horizontally where the key::
```

Comment:
@lyrixx What does the "where the key" part of the sentence refer to?

Reply:
Oups, I don't know what I wanted to say. It's a type that should be removed.
Do you want me to do a PR?

Reply:
That would be great. :) Or if you leave a comment here about what sentence should be placed there, I can do that too.

Reply:
Oh, it's already fixed by Javier in a5e1d81ce5ac8c463f782456baa55b5e40af389d :heart:

Reply:
😆 I should have checked. Thanks @javiereguiluz! 👍